src/hotspot/share/runtime/arguments.cpp

```diff
@@ -4069,9 +4069,6 @@ jint Arguments::apply_ergo() {
 #ifdef COMPILER1
       || !UseFastLocking
 #endif // COMPILER1
-#if INCLUDE_JVMCI
-      || !JVMCIUseFastLocking
```

Comment:
I was a bit concerned this subtly changes the observable VM behavior by JVMCI compilers, but it looks like the flag is already `develop`, and so cannot really affect release binaries.